cityofaustin/janis
# 4392 nav2: Header and Themes Buttons

Zenhub Issue: 🤝 Global nav v2 dev criteria#4392: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4392

Deployed: https://janis-v3-4392-nav-v2.netlify.com/

The original issue was broken into two parts that could be rolled out without regression.
- 1st (this PR): Header and Themes Buttons 
- 2nd: Adding the new Mega nav with theme specific dropdown menus. 
  - Zenhub issue`New Mega Nav#4648`: https://github.com/cityofaustin/techstack/issues/4648

# Description








# Types of changes



# How can this be tested?

From: https://janis-v3-4392-nav-v2.netlify.com/
- click on the search button, did the search field open?
- search something in the searchbar and test different ways to execute the search...
  - 1. Type a term and press enter, did it search the term?
  - 2. Type a term and press the search button, did it search the term?
  - 3. Type and term, hit tab, and hit enter, did it search the term? 
- From the home page
  - hover over the new themes nav, see a pretty blue pill pop up. 
- Check that desktop and mobile styles matches design.

# Checklist:
